### PR TITLE
Update munin blogpost

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
This pull request adds loss, epoch and logs to blogpost about minun

@rlrs må jeg tilføje linket her?

### CODE REVIEW

**Comment by rlrs** [comment]
I made a "report" on wandb which is a bit nicer, but maybe hosting it somewhere more permanent than wandb makes sense.
https://api.wandb.ai/links/rlrs/wbqovk26

**Comment by KennethEnevoldsen** [comment]
En pdf er en mulighed?

**Comment by peterbjorgensen** [comment]
Can we merge or close this?

**Comment by rlrs** [comment]
Sure, the easiest is to just put a link to the wandb run, but it might not be the best way.

File: docs/blog/posts/munin-7b-alpha.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ model. It has been pre-trained on the [Danish Gigaword](https://gigaword.dk/) da
 which has been instrumental in training various Danish BERT-style models.
 
 <!-- more -->
+The model has been trained for one epoch over the dataset and ends up with a loss 1.27 on the Danish Gigaword. See more the model training logs [here](missing).
```
Comment (KennethEnevoldsen) [inline docs/blog/posts/munin-7b-alpha.md:18]:
```suggestion
The model has been trained for one epoch over the dataset and ends up with a loss 1.27 on the Danish Gigaword. See more the model training logs [here](https://api.wandb.ai/links/rlrs/wbqovk26).
```

**Pull request merged**